test/src/RedisMutex.test.ts

```diff
@@ -71,7 +71,24 @@ describe('Mutex', () => {
   it('should refresh lock every refreshInterval ms until release', async () => {
     const mutex = new Mutex(client, 'key', timeoutOptions)
     await mutex.acquire()
-    await delay(100)
+    await delay(400)
```

Comment:
previous delay appears to be wrong, lock wouldn't expire either way before 300 msecs